css/style.css

```diff
@@ -81,7 +81,7 @@ body {
   text-decoration: none;
   color: #fff;
   font-style: normal;
-  font-size: 2.1rem;
+  font-size: 2.14rem;
```

Comment:
It'd be weird for designer to pick font size of `21.4px` 😅

Reply:
corrected